unit-tests/game-api-tests.js

```diff
@@ -0,0 +1,32 @@
+describe('gameApiTest', function(){
+
+    var gameApi;
+
+    beforeEach(function(){
+        module('tombola.noughtAndCrosses.services');
+        inject(function($injector){
+            gameApi = $injector.get('gameApi');
```

Comment:
This will use the actual web proxy - tying you in to he real implementation, which is v. bad. 
I'll pop over after I've finished with the grads to help.